khmer/khmer_args.py

```diff
@@ -149,6 +311,7 @@ def calculate_tablesize(args, hashtype, multiplier=1.0):
 
 
 def create_nodegraph(args, ksize=None, multiplier=1.0):
+    args = check_fp_rate(args)
```

Comment:
On Wed, Aug 05, 2015 at 10:03:57AM -0700, Jake Fenton wrote:
> > @@ -149,6 +311,7 @@ def calculate_tablesize(args, hashtype, multiplier=1.0):
> >  
> >  
> >  def create_nodegraph(args, ksize=None, multiplier=1.0):
> > +    args = check_fp_rate(args)
> 
> I'm conflicted on how to manage the fp rate setting--Currently, all scripts default to an fp of 0.1 but that can be overidden via the --fp-rate arg, which is in the args object and is passed along.
> 
> If we want to actually pass in a default rate we'd have to add an arg to at least both create functions--Which isn't necessarily bad, but I'd like to avoid it if possible.
> 
> We could have the scripts instead modify the fp_rate attribute of the args... Though I don't feel like that's a good solution either

There's science in them thar hills, so we need something like the following:

* if users supply fp-rate, assume they know what they are doing;
* otherwise, default to the one specified by the script;
* if one isn't specified by the script, default to the global one;

So, yes, we need an arg added to both create functions; there's no avoiding it.